lang/en/tool_objectfs.php

```diff
@@ -115,4 +125,4 @@
 $string['settings:testingheader'] = 'Test Settings';
 $string['settings:testingdescr'] = 'This setting is mainly for testing purposes and introduces overhead to check the location.';
 
-$string['settings:error:numeric'] = 'Please enter a number which is greater than or equal 0.';
\ No newline at end of file
+$string['settings:error:numeric'] = 'Please enter a number which is greater than or equal 0.';
```

Comment:
Probably you don't need to change this as part of this pull request.

Reply:
new line has been removed